TradeItIosTicketSDK2/TradeItLauncher.swift

```diff
@@ -332,4 +332,23 @@ protocol OAuthCompletionListener {
         )
     }
 
+    public func launchTransactions(
+        fromViewController viewController: UIViewController,
+        forLinkedBrokerAccount linkedBrokerAccount: TradeItLinkedBrokerAccount
+        ) {
+        deviceManager.authenticateUserWithTouchId(
+            onSuccess: {
+                let navController = self.viewControllerProvider.provideNavigationController(withRootViewStoryboardId: .transactionsView)
+
+                guard let transactionsViewController = navController.viewControllers.last as? TradeItTransactionsViewController else { return }
+                transactionsViewController.linkedBrokerAccount = linkedBrokerAccount
+                transactionsViewController.enableThemeOnLoad = false
+                transactionsViewController.view.backgroundColor = UIColor.tradeItlightGreyHeaderBackgroundColor
```

Comment:
How come the theme is disabled?

Reply:
Enabling the theme would set some specific styling on the headers/table that doesn't look like the mocks.

Reply:
OK - we should address this later if the mocks were from us. We need to conform to a specific set of styles because it makes maintaining the project harder. If they wanted to change the way headers look, then we should do it throughout the SDK with the theme configurator. Next time we get a mock that doesn't conform, let's raise it up and talk about it.